### DISCUSSION TITLE
automatiser un va et vient virtuel avec 2 inter zigbee - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#17] [Comments]
# automatiser un va et vient virtuel avec 2 inter zigbee - Haade.fr

nous allons automatiser simplement grâce à blueprint et homeassistant un va et vient virtuel entre 2 interrupteurs simple voie zigbee

https://haade.fr/fr/blog/automatiser-un-va-et-vient-virtuel-avec-blueprint-et-homeassistant

**Comment by yahtala75** [comment]
Bonjour Nico,

Merci pour ton article. Je cherchais justement à faire ça avec des interrupteurs Zigbee MOES 1 Gang - neutre optionnel. (Je n’ai pas de câble Neutre mais j’ai déjà deux câbles navettes)
Si j’ai bien compris, il faut que je connecte le L de l’interrupteur A avec la phase côté disjoncteur et le L de l’interrupteur B avec la phase côté lampe. Ensuite je fais un pont entre les L1 des deux interrupteurs et un autre pont entre les N des deux interrupteurs.
J’ai alors une question, si le serveur HA est down, est-ce que les interrupteurs sont fonctionnel ? (Hors va-et-vient évidemment)
Merci

**Reply by haade-administrator** [reply]
Salut oui avec la phase en entrée et sortie ampoule l'interrupteur fonctionnera toujours même home assistant hs

**Comment by Chriskito30** [comment]
Bonjour,

Pouvez-vous éclaircir cette phrase s'il vous plaît ? "et faire un pont du neutre rouge afin d’apporter le neutre au second interrupteur"
J'ai l'impression qu'il y a une coquille entre vos explications ainsi que le 2ieme schéma 🤔
